vue-project-stub-no-pwa/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "hello-world",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit --testPathPattern=."
+  },
+  "dependencies": {
+    "@mdi/font": "^4.5.95",
+    "axios": "^0.18.0",
+    "core-js": "^3.4.4",
+    "css-loader": "^0.28.11",
+    "regenerator-runtime": "^0.13.3",
+    "sbc-common-components": "2.0.30",
+    "vue": "^2.6.10",
+    "vuex-class": "^0.3.2",
+    "vue-class-component": "^7.0.2",
+    "vue-property-decorator": "^8.3.0",
```

Comment:
"vue-class-component" and "vue-property-decorator" can be moved to devDependencies (which makes the dist smaller). Also perhaps "vuex-class" can me moved?

(Please apply these changes to the create business project also.)

Reply:
Sounds good, will do!